docs/roadmap.md

```diff
@@ -34,13 +34,13 @@ See also [this issue](https://github.com/daostack/daostack/issues/14)
 
 # UI
```

Comment:
i think the roadmap.md should be updated and revised.
i suggest to remove this for now ,open an issue for that and pr a new roadmap doc

Reply:
removed link to `roadmap.md` in https://github.com/daostack/daostack/pull/270
+ added issue
in the meantime, until it's updated, fixing the typos doesn't hurt.